docs/api/grpc/overview.md

```diff
@@ -1,16 +1,154 @@
 ---
-title: gRPC API overview
+title: Using gRPC
+sidebar_label: Using gRPC
 hide_title: false
+description: Learn how to use the gPRC API.
+vega_network: TESTNET
 ---
-gRPC is a low-latency, highly efficient API for interacting with the network and querying for and filtering data.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Core node APIs
-Core nodes are responsible for ensuring the consensus rules are met and that a consistent view of the network is seen. They present endpoints that give access to the state of the network (block time, block height etc), allow transactions to be submitted to the network and to subscribe to event streams so that changes of internal state can be seen.
+gRPC is a low-latency, highly efficient communication method that allows a client and a server to send data over HTTP. Vega supports a gRPC API for interacting with the network and querying for and filtering data. 
 
-- [Core](vega/api/v1/core.proto.mdx): Submit transactions to the network and retrieve basic information about the node and network such as the block height, app version, number of peers, etc.
-- [Core state](vega/api/v1/corestate.proto): Request information about the present ‘state’ of the Vega node. Importantly, the core node only maintains enough state information such that it can process incoming transactions; it keeps no history beyond what is required for that function. You can request current lists of accounts, parties, proposals etc., which are held in memory by the core node.
+## gRPC API and Protocol Buffers
+gRPC uses Protocol Buffers (protobufs) as its underlying data definition. Protobuf messages are language-neutral data definitions that are used to generate language-specific data structures. Code generation of a protobuf message also generates functions to serialise each message so that they can be sent through gRPC to each RPC service endpoint.
 
-## Data node API
+This means that Vega's gRPC API is fully defined by its protobuf definitions, and so the protobuf definitions themselves provide complete documentation.
+
+## Relationship to REST API
+The buf ecosystem contains plugins that also allow automatic generation of REST API endpoints from the protobuf definitions. The result is that Vega's gRPC and REST API match exactly in both structure and functionality.
+
+As REST is a more familiar way to interact with a product than gRPC, experiencing Vega first through the REST API and using the REST documentation may help ease an initial integration, without any loss of functionality. Migrating later to using gRPC is then trivial since the mapping between REST and gRPC is 1-to-1.
```

Comment:
```suggestion
If you are new to interacting with gRPC, using REST may be a more familiar way to interact with the APIs. Experiencing Vega first through the REST API and using the REST documentation may help ease an initial integration, without any loss of functionality. Migrating later to using gRPC is then trivial since the mapping between REST and gRPC is 1-to-1.
```